undertow/src/main/resources/org/wildfly/extension/undertow/LocalDescriptions.properties

```diff
@@ -312,6 +312,17 @@ undertow.setting.jsp.x-powered-by=Enable advertising the Jakarta Server Pages en
 undertow.setting.jsp.display-source-fragment=When a runtime error occurs, attempts to display corresponding Jakarta Server Pages source fragment
 undertow.setting.jsp.optimize-scriptlets=If Jakarta Server Pages scriptlets should be optimised to remove string concatenation
 
+undertow.setting.affinity-cookie=Session cookie configuration
```

Comment:
We have now two resources with the same description, I suggest to changes this to `Affinity cookie configuration`.  The same for description of `add` and `remove` operations bellow.

Reply:
@simkam Good catch, this fell off my todo list, these are meant to be different as you suggest. Also, the comment property below is never used as that was dropped altogether by the RFC. Fixed both now. Thanks.